error.go

```diff
@@ -0,0 +1,19 @@
+package tprstorage
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microstorage"
+	"github.com/juju/errgo"
+)
+
+var invalidConfigError = microerror.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return errgo.Cause(err) == invalidConfigError
+}
+
+// Errors from microstorage must be reused in order to match microstorage error
+// matchers. This is required to fulfil the interface and pass storagetest.
+
+var notFoundError = microstorage.NotFoundError
```

Comment:
This implements `microstorage.Storage`. In the `Storage` interface we should specify which errors are returned and when. This should also be well 

I don't see a clever way to work around that ATM.

I'm curious what problems that may cause.